packages/wp-now/src/start-server.ts

```diff
@@ -95,5 +96,22 @@ export async function startServer(options: WPNowOptions = {}) {
 
 	app.listen(port, () => {
 		console.log(`Server running at http://127.0.0.1:${port}/`);
+		const porturl = `http://127.0.0.1:${port}`;
```

Comment:
also it may make sense simply to call it `url` since the port is part of the URL. that's going to flow more naturally below when we spawn `open ${url}`